dev-python/av/av-7.0.1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Pythonic bindings for FFmpeg's libraries."
+HOMEPAGE="https://github.com/mikeboers/PyAV https://pypi.org/project/av/"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="test"
+
+RDEPEND=""
+DEPEND="${REDEPEND}
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	test? (
+		dev-python/nose[${PYTHON_USEDEP}]
+		dev-python/pytest[${PYTHON_USEDEP}]
+	)"
+
+python_test() {
+	nosetests --verbose || die
+	py.test -v -v || die
+}
```

Comment:
Can this be converted using the `distutils_enable_tests()` as documented in https://devmanual.gentoo.org/eclass-reference/distutils-r1.eclass?